app/components/atoms/paragraph/Paragraph.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import styled, {css} from 'styled-components';
+
+export interface ParagraphVariant {
+  height: 'small' | 'large';
+  weight: 'normal' | 'bold'
+}
+
+const heights = {
+  small: css`
```

Comment:
there is no `normal` height?

Reply:
normal is a valid value for weight, not for height :)